<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDBMaintenance</class>
 <widget class="QFrame" name="CDBMaintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库维护</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(D:/WinParkUI/debug/Image/NewIcon/CommonMiddleBG-normal.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>85</y>
     <width>651</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>出入记录自动删除设置</string>
    </property>
    <widget class="QCheckBox" name="chxDay2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>自动删除过期出入记录</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>出入记录保留天数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spDay2">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>清除出入记录</string>
    </property>
    <widget class="QPushButton" name="btnClear2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>166</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDateEdit" name="date2">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>之前的数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>530</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>备份数据</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>21</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>保存路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtBackupPath">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnBackupPath">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>226</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QCheckBox" name="chxBackup">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>同时备份进出图像数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBackup">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>开始备份</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>续费记录自动删除设置</string>
    </property>
    <widget class="QCheckBox" name="chxDay1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>自动删除过期续费记录</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>续费记录保留天数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spDay1">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>清除系统日志</string>
    </property>
    <widget class="QPushButton" name="btnClear3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>166</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QDateEdit" name="date3">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>之前的数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>恢复数据</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>文件路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtRecoveryPath">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRecoveyPath">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QCheckBox" name="chxOverwrite">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>相同数据覆盖</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRecovery">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>开始恢复</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>进出图像自动删除设置</string>
    </property>
    <widget class="QCheckBox" name="chxDay4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>自动删除过期图像数据</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>图像数据保留天数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spDay4">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>数据自动备份记录</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>开始备份时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>下次备份时间：</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>145</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTimeEdit" name="timeAuto">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chxAuto1">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>打开自动数据备份</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chxAuto2">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>同时备份进出图像数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QDateEdit" name="dtNext">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>188</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>清除续费纪录</string>
    </property>
    <widget class="QPushButton" name="btnClear1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>166</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateEdit" name="date1">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>之前的数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>清除图像数据</string>
    </property>
    <widget class="QPushButton" name="btnClear4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>166</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QDateEdit" name="date4">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>之前的数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="title">
     <string>系统日志自动删除设置</string>
    </property>
    <widget class="QCheckBox" name="chxDay3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>自动删除过期日志记录</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>日志记录保留天数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spDay3">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>每天数据删除时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeDelete"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>7</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
